modules/ROOT/pages/configurable-archive.adoc

```diff
@@ -27,11 +27,11 @@ In general we recommended that you set all attributes to *true* or all to *false
 * To archive a loop activity each time it changes state, set `bonita.tenant.archive.SLoopActivityInstance` to true.
 * To archive a multi-instances activity each time it changes state, set `bonita.tenant.archive.SMultiInstanceActivityInstance` to true.
 * To archive a call activity each time it changes state, set `bonita.tenant.archive.SCallActivityInstance` to true.
-* To archive a gateway each time it changes state, set `bonita.tenant.archive.SGatewayInstance` to true. This information is not available in Bonita Portal.
+* To archive a gateway each time it changes state, set `bonita.tenant.archive.SGatewayInstance` to true. This information is not available in the Bonita Applications.
```

Comment:
not available in the Bonita Applications. => not available in Bonita Applications.

To be coherent with line 33.

Reply:
Done